oneutil/etl/databento.py

```diff
@@ -54,3 +60,130 @@ def get_df_from_s3(
 
     # Return the DataFrame
     return df
+
+
+def get_files_in_s3_folder(
+        foldername: str,
```

Comment:
let's make it more general purpose so it doesn't have to mean a top-level folder but can also be a path.

e.g. if I have bucket/data/nasdaq/daily your code can already work to retrieve the filenames. so we can be more generic

```suggestion
def get_files_in_s3_path(
        path: str,
```